<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDial" name="dialeye1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>120</number>
    </property>
   </widget>
   <widget class="QDial" name="dialeye2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>120</number>
    </property>
   </widget>
   <widget class="QDial" name="dialeye4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>220</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>120</number>
    </property>
   </widget>
   <widget class="QDial" name="dialeye3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>120</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roman</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Left eye Vertical</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roman</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Left eye Horizontal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>330</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roman</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Right eye Horizontal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roman</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Right eye Vertical</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS UI Gothic</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CONTROL</string>
    </property>
   </widget>
   <widget class="QDial" name="dialeye33">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>440</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>120</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>510</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roman</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Right eye Vertical 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>690</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roman</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Right eye Horizontal 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>680</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roman</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Left eye Horizontal 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>510</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roman</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Left eye Vertical 2</string>
    </property>
   </widget>
   <widget class="QDial" name="dialeye11">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>440</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>120</number>
    </property>
   </widget>
   <widget class="QDial" name="dialeye22">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>620</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>120</number>
    </property>
   </widget>
   <widget class="QDial" name="dialeye44">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>620</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>1011</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS UI Gothic</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SEQUENCE</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton1">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>440</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Position 1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>470</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Position 2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>500</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Position 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSave">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>470</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPlay">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>570</y>
      <width>251</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niagara Engraved</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reproduce</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLEV">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelREV">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>110</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLEH">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>320</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelREH">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>320</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLEV2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>500</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLEH2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>680</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelREV2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>500</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelREH2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>680</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>923</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
